frontend/src/metabase/query_builder/components/NativeQueryEditor/NativeQueryEditor.tsx

```diff
@@ -505,12 +538,7 @@ export class NativeQueryEditor extends Component<
           }
 
           // transform results into what ACE expects
-          const resultsForAce = results?.map(([name, meta]) => ({
-            name: name,
-            value: name,
-            meta: meta,
-          }));
-          callback(null, resultsForAce || []);
+          callback(null, resultsForAce(results));
```

Comment:
I'd rename `resultsForAce` to highlight it's a method, maybe `getResultsForAce` or `prepareResultsForAce`